api/server/cluster.go

```diff
@@ -711,3 +712,8 @@ func handleResourceType(resource string) (api.ResourceType, error) {
 		return api.ResourceType_RESOURCE_TYPE_NONE, fmt.Errorf("Invalid resource type")
 	}
 }
+
+func (c *clusterApi) RegisterSecretManager(sec secrets.SecretManager) *secrets.Manager {
```

Comment:
is this right place to add RegisterSecretManager()?

Reply:
Yes since it is a receiver for (*clusterAPI)

Reply:
Why does this return a secrets manager? For possible chaining?